src/validation.ts

```diff
@@ -23,6 +29,65 @@ export function assertAndGetScraperEnvVars() {
     scraperApiJwt: process.env.SCRAPER_API_JWT
   };
 }
+
+export function assertAndGetOwnerWallet() {
+  assert(process.env.INFURA_API_KEY, "Env var INFURA_API_KEY missing");
+  assert(
+    process.env.OWNER_PK || process.env.OWNER_MNEMONIC,
+    "Either OWNER_PK or OWNER_MNEMONIC need to be set"
+  );
+
+  return {
+    infuraApiKey: process.env.INFURA_API_KEY,
+    ownerWallet: process.env.OWNER_PK
+      ? new Wallet(process.env.OWNER_PK)
+      : Wallet.fromMnemonic(process.env.OWNER_MNEMONIC || "")
+  };
```

Comment:
Any reason not to initialize the provider and signer objects here? Right now, it seems we initialize the Wallet, but we require the caller to build the provider and signers from these building blocks.

Reply:
Yea no reason. I connected the provider directly to the wallet [here](https://github.com/across-protocol/merkle-distributor/pull/3/commits/03120899f65a4726ad29a65d8690c26964370215)